vnext/Desktop/module.g.cpp

```diff
@@ -30,11 +32,26 @@ void* __stdcall winrt_get_activation_factory([[maybe_unused]] std::wstring_view
         return std::equal(left.rbegin(), left.rend(), right.rbegin(), right.rend());
     };
 
+    if (requal(name, L"Microsoft.ReactNative.ReactDispatcherHelper"))
```

Comment:
Just checking if this is intentional as part of the upgrade or perhaps another change? 
Or did someone miss running the generator and checking this file in?

Reply:
Yes, this is an intentional change: the file generated by C++/WinRT is not compilable by C++. We must change the file and copy it locally to compile.